ui/src/utils/verification-utils.js

```diff
@@ -1,22 +1,21 @@
-import {
-  verifyCredential /*, downloadRevocationList*/,
-} from "@sunbird-rc/verification-sdk";
-import { resolveDid } from "./did-utils";
-
-let revocationList = [];
-
 const verify = async (credential) => {
-  let resolutionResult = await resolveDid(
-    credential?.proof?.verificationMethod
-  );
-  if (resolutionResult.didResolutionMetadata.error) {
-    throw new Error(resolutionResult.didResolutionMetadata.error);
-  }
-  let issuerDID = resolutionResult.didDocument; /*
-    let revocationUrl = "http://localhost:3000/credentials/revocation-list";*/
-  revocationList =
-    /*await downloadRevocationList(issuerDID.id, revocationUrl)*/ [];
-  return await verifyCredential(issuerDID, credential, revocationList);
+  return await verifyCredentialOnline(credential);
 };
 
+const verifyCredentialOnline = (credential) => {
+  return new Promise(function (resolve, reject) {
+    const xhttp = new XMLHttpRequest();
+    xhttp.open("POST", `${window._env_.VERIFY_SERVICE_API_URL}/verify-vc`, true);
```

Comment:
why we need endpoint name as `verify-vc`, it can be simple `/credential`
as we already request mapping as `v1/verify` so full url will be `v1/verify/credential`